_includes/404.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+
+<!-- 404 page by https://codepen.io/gxash/pen/rrJYwm -->
+
+<html lang="en">
+
+{% include head.html %}
+
+<body class="bootstrap">
+
+  <div class="container">
+
+    <div class="wrapper p-5 text-center">
```

Comment:
Why does this file have a giant built-in svg?

Reply:
![image](https://github.com/randovania/randovania.github.io/assets/2979303/7f98939d-a03c-4e81-a355-c5c3a8fd3bad)
idk it came that way and it's pretty lol

Reply:
I would at least externalize the svg out TBH. rather than have a huge blob.

Reply:
feel free! i have no idea how to do that and never really had any intention of touching this page